sw/device/tests/dif/dif_gpio_smoketest.c

```diff
@@ -15,6 +15,11 @@
 static dif_gpio_t gpio;
 const test_config_t kTestConfig;
 
+// A known pattern written to GPIOs.
```

Comment:
Would you mind adding a comment that says sthg like "these are used as-is when the test is run on the FPGA but are randomized in DV" (and maybe mention the name of the file if it makes sense)?

Can you also use the `/** */` syntax for the sake of consistency?

Reply:
Done.